extensions/sql-bindings/src/extension.ts

```diff
@@ -23,8 +23,8 @@ export async function activate(context: vscode.ExtensionContext): Promise<IExten
 		createAzureFunction: async (): Promise<void> => {
 			return createAzureFunction();
 		},
-		promptForBindingType: async (funcName?: string): Promise<BindingType | undefined> => {
-			return promptForBindingType(funcName);
+		promptForBindingType: async (objectType?: string, funcName?: string): Promise<BindingType | undefined> => {
```

Comment:
Currently, I can have this optional due to its use in addSQLBinding prompt where it would not be needed as of yet. Will be seeing the flow for that in a separate PR, which should not have been changed in regard to this PR.

Reply:
Couldn't you just specify it as Table for now and then when you get to adding view support there change it at that point?

Reply:
Specify the table for now to add SQL binding? It will only be asked if a user chooses to use to create a new SQLConnectionString different from that of the one that is already there for the Azure function.

Add SQL Binding flow is quite different from the Create Azure Function with SQL Binding, since a user can still use view if a user selects input binding, same SQLConnectionString, and enters the object name that could be a view.

Reply:
For now, will keep it as is. As the Add SQL binding quickpick flow does not change from this, I will add it as a follow-up priority for views functionality.

Reply:
Alright, although see my other comment for why I am specifically calling this out. When you have behavior that isn't clearly defined then you open yourself up to bugs down the line because there isn't a clear expectation what to expect so people making changes may not adhere to what the current behavior is. Making those expectations clear from the start helps avoid that and makes it easier to validate that what you've written is doing exactly what you intended.